### ISSUE TITLE
Have extra delay before executing command

### ISSUE THREAD

**Issue by SR-G** [SR-G/sleep-on-lan#48]
A GOLANG "defer" was already in place, but due to the internal GOLANG framework used here for HTTP handlers and depending on the computer being used, HTTP answers could be skipped before sleep being executed.

Hence a new possible configuration : 

This is now activated by default with a 500ms value

**Comment by SR-G** [comment]
Now this should be always answered. 

```curl http://192.168.8.3:8009/sleep
<?xml version="1.0" encoding="UTF-8"?>
<result>
  <operation>sleep</operation>
  <successful>true</successful>
</result>%      
```

**Comment by SR-G** [comment]
See configuration in case of need to override.

```
    "DelayBeforeCommands": {
        "Active": true,
        "Delay": "500ms"
    }
```

**Comment by SR-G** [comment]
Included in 1.1.1-RELEASE
